contrib/ros/envs/sawyer/pick_and_place_env.py

```diff
@@ -2,40 +2,95 @@
 Pick-and-place task for the sawyer robot
 """
 
+import numpy as np
 from rllab.core.serializable import Serializable
 
-from contrib.ros.envs import sawyer_env
+from contrib.ros.envs.ros_env import RosEnv
 from contrib.ros.robots.sawyer import Sawyer
 
-INITIAL_MODEL_POS = {
-    'table0': [0.75, 0.0, 0.0],
-    'object0': [0.5725, 0.1265, 0.80]
-}
 
-INITIAL_ROBOT_JOINT_POS = {
-    'right_j0': -0.041662954890248294,
-    'right_j1': -1.0258291091425074,
-    'right_j2': 0.0293680414401436,
-    'right_j3': 2.17518162913313,
-    'right_j4': -0.06703022873354225,
-    'right_j5': 0.3968371433926965,
-    'right_j6': 1.7659649178699421,
-}
+class PickAndPlaceEnv(RosEnv, Serializable):
+    def __init__(self,
+                 initial_goal,
+                 task_obj_mgr,
+                 sparse_reward=True,
+                 simulated=False):
+        Serializable.quick_init(self, locals())
 
+        self._distance_threshold = 0.05
+        self._target_range = 0.15
```

Comment:
It would be nice to be able to set these parameters somewhere. Maybe you can add them to the constructor args with these default values.